R/adjust.R

```diff
@@ -82,6 +83,11 @@ adjust_infection_to_report <- function(infections, delay_defs,
                                        reporting_model, reporting_effect,
                                        type = "sample",
                                        truncate_future = TRUE) {
+  deprecate_warn(
+    "1.5.0",
+    "adjust_infection_to_report()",
+    "The function will be removed completely in version 2.0.0."
```

Comment:
this should be in a named argument `details` (see failing tests).

Reply:
My bad. I've fixed it now.